backlog/models.py

```diff
@@ -312,7 +312,10 @@ class Meta:
 class AssemblyAnnotationJob(models.Model):
     class Meta:
         db_table = 'AssemblyAnnotationJob'
-        app_label='backlog'
+        app_label = 'backlog'
 
     assembly = models.ForeignKey(Assembly, on_delete=models.DO_NOTHING, related_name='assemblyannotationjobs')
     annotation_job = models.ForeignKey(AnnotationJob, on_delete=models.CASCADE)
+    protein_db = models.BooleanField(
+        "Boolean field to mark was assembly from Job added to protein DB or not",
```

Comment:
May I suggest a different working: "True if the linked assembly was added to the protein DB"?